src/ILLink.Shared/ILLink.xsd

```diff
@@ -0,0 +1,193 @@
+		</xsd:complexType>
+	</xsd:element>
+
+	<xsd:element name="argument">
+		<xsd:complexType mixed="true">
+			<xsd:choice minOccurs="0" maxOccurs="1">
+				<xsd:element name="argument">
+					<xsd:complexType>
+						<xsd:simpleContent>
+							<xsd:extension base="xsd:string">
+								<xsd:attribute name="type" />
+							</xsd:extension>
+						</xsd:simpleContent>
+					</xsd:complexType>
+				</xsd:element>
+			</xsd:choice>
+			<xsd:attribute name="type" />
+		</xsd:complexType>
+	</xsd:element>
+	
+	<xsd:element name="attribute">
+		<xsd:complexType>
+			<xsd:choice maxOccurs="unbounded" minOccurs="0">
+				<xsd:element ref="argument" />
+				<xsd:element minOccurs="0" maxOccurs="unbounded" name="property">
+					<xsd:complexType>
+						<xsd:simpleContent>
+							<xsd:extension base="xsd:string">
+								<xsd:attribute name="name" />
+							</xsd:extension>
+						</xsd:simpleContent>
+					</xsd:complexType>
+				</xsd:element>
+			</xsd:choice>
+			<xsd:attribute name="internal" type="xsd:string" />
+			<xsd:attribute name="fullname" type="xsd:string" />
+			<xsd:attribute name="assembly" type="xsd:string" />
+		</xsd:complexType>
+	</xsd:element>
+
+	<xsd:element name="namespace">
+		<xsd:complexType>
+			<xsd:attribute name="fullname"/>
+		</xsd:complexType>
+	</xsd:element>
+	
+	<xsd:element name="assembly">
+		<xsd:complexType>
+			<xsd:choice minOccurs="0" maxOccurs="unbounded">
+				<xsd:element minOccurs="0" maxOccurs="unbounded" ref="type" />
+				<xsd:element minOccurs="0" maxOccurs="unbounded" ref="attribute" />
+				<xsd:element minOccurs="0" maxOccurs="unbounded" ref="resource" />
+				<xsd:element minOccurs="0" maxOccurs="unbounded" ref="namespace" />
+			</xsd:choice>
+			<xsd:attribute name="fullname" type="xsd:string" use="required" />
+			<xsd:attribute name="feature" />
+			<xsd:attribute name="featurevalue" type="xsd:boolean" />
+			<xsd:attribute name="featuredefault" type="xsd:boolean" />
+			<xsd:attribute name="preserve" type="preservevalues" />
+		</xsd:complexType>
+	</xsd:element>
+	
+	<xsd:simpleType name="preservevalues">
+		<xsd:restriction base="xsd:string">
+			<xsd:enumeration value="all" />
+			<xsd:enumeration value="none" />
+			<xsd:enumeration value="fields" />
+			<xsd:enumeration value="methods" />
+		</xsd:restriction>
+	</xsd:simpleType>
+
+	<xsd:simpleType name="accessorvalues">
+		<xsd:restriction base="xsd:string">
+			<xsd:enumeration value="all" />
+			<xsd:enumeration value="get" />
+			<xsd:enumeration value="set" />
+		</xsd:restriction>
+	</xsd:simpleType>
+
+	<xsd:element name="type">
+		<xsd:complexType>
+			<xsd:choice minOccurs="0" maxOccurs="unbounded">
+				<xsd:element ref="event" />
+				<xsd:element ref="method" />
+				<xsd:element ref="attribute" />
+				<xsd:element ref="field" />
+				<xsd:element ref="property" />
+				<xsd:element ref="type" />
+			</xsd:choice>
+			<xsd:attribute name="fullname" type="xsd:string" />
+			<xsd:attribute name="name" type="xsd:string" />
```

Comment:
Do we support `name` on `type`?

Reply:
The documentation has `name` supported on nested `type`s, but not on top level.

Reply:
What does the code do?

Reply:
Looks like it also allows name on nested types but not on top level.

Reply:
`protected virtual void ProcessTypes (AssemblyDefinition assembly, XPathNavigator nav, bool warnOnUnresolvedTypes)` in ProcessLinkerXmlBase.cs gets the `fullname` attribute, but in LinkAttributesParser.cs it will look for name, and doesn't look for `fullname`